doc/api/buffer.md

```diff
@@ -941,6 +941,28 @@ added: v0.9.1
 Returns `true` if `encoding` contains a supported character encoding, or `false`
 otherwise.
 
+### Class Method: Buffer.of(...items)
+<!-- YAML
+added: 6.0.0
+-->
+
+* `...items` {number[]} A list of values for buffer initialisation
+* Returns: {Buffer}
+
+Returns a `Buffer` from the list of values. Alternative to
+`Buffer.from` that bypasses the `Array` constructor and
+several internal setup calls.
+
+```js
+Buffer.from(1, 2, 3, 4).equals(Buffer.from([1, 2, 3, 4]))
```

Comment:
The first call should be `Buffer.of()`. End the line with a semicolon. Also, it would be helpful to have the return value of `.equals()` as a comment.